.changes/allow-back-forward-gesture.md

```diff
@@ -0,0 +1,11 @@
+---
+"wry": minor
+---
+
+Add the ability to navigate with swipe gesture
+
+This switch is for WebView on macOS, enabling us to swipe horizontally to trigger backward and forward page navigation.
+
+This commit also includes a worked example for it.
+
+For details, see <https://developer.apple.com/documentation/webkit/wkwebview/1414995-allowsbackforwardnavigationgestu>
```

Comment:
On macOS, add an API to enable or disable backward and forward navigation gestures.